### DISCUSSION TITLE
Is it possible to install packer manager in Lazy vim?

### DISCUSSION THREAD

**Discussion by tonmoy998** [LazyVim/LazyVim#711] [Q&A] [answered]
Hey I am new to Lazy vim and lua.In earlier I used vim.I was trying to install packer .

**Comment by folke** [comment]
You want to install packer in a distro built around lazy.nvim?

**Reply by tonmoy998** [reply]
yes

**Reply by tonmoy998** [reply]
Hey,Can you please give me an example of installing emmet-vim using  lazy?

**Comment by tonmoy998** [comment, marked as answer]
Is it right way to install?

![Screenshot_20230506_230624](https://user-images.githubusercontent.com/74363531/236637589-b96c81bb-a203-46c2-a498-0d63ebf7bbd5.png)

**Reply by dpetka2001** [reply]
You don't need `version = false`, since it's already the default for `lazy.nvim`. You also need either `opts = {}` or `config = true` to call plugin's setup function.

**Reply by dpetka2001** [reply]
FYI there's also `emmet-ls` which you can install via `mason`. I don't know how different it is from `emmet-vim`, but I've been using it so far without any problems.

**Reply by tonmoy998** [reply]
Thank you so much.I will explore ;)

**Reply by tonmoy998** [reply]
thank you! I successfully installed.
